hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -1510,12 +1510,20 @@ public void performTableServices(Option<String> inFlightInstantTimestamp, boolea
       allTableServicesExecutedSuccessfullyOrSkipped = false;
       throw e;
     } finally {
+      String tableName = writeClient.getConfig().getTableName();
+      boolean tableNameExists = StringUtils.nonEmpty(tableName);
+      String executionDurationMetricName = tableNameExists
+              ? String.format("%s.%s", tableName, HoodieMetadataMetrics.TABLE_SERVICE_EXECUTION_DURATION)
```

Comment:
Does other metric reporter require table name as the first component in the metric name (is it compatible to all metric reporters)?

Reply:
I don't see other metrics reporter require `tableName` in the dimensions so they should be ok